src/api/api.js

```diff
@@ -0,0 +1,7 @@
+const URL = 'http://www.omdbapi.com/?apikey=e6b7e90f';
```

Comment:
You should change the URL to https. Because GitHub doesn't allow requests to http.